tests/factories.py

```diff
@@ -243,6 +248,145 @@ def create_release_request(workspace, user=None, **kwargs):
     return release_request
 
 
+def create_request_at_state(
```

Comment:
I realise this is going to mean a lot of search-replace, but we renamed the enum from state to status, so....

```suggestion
def create_request_at_status(
```